src/umpire/resource/MemoryResourceRegistry.cpp

```diff
@@ -19,9 +69,140 @@ MemoryResourceRegistry& MemoryResourceRegistry::getInstance() noexcept
   return resource_registry;
 }
 
+const std::vector<std::string>&
+MemoryResourceRegistry::getResourceNames() noexcept
```

Comment:
This is gonna give me a merge conflict D:

Reply:
Lol that's ok though

Reply:
😄 you can just refactor your method in ResourceManager, shouldn't conflict (this is in the MemoryResourceRegistry, which you weren't editing)